tests/test_serializer.py

```diff
@@ -384,10 +384,16 @@ def create(self, validated_data):
     def test_not_required_output_for_allow_null_field(self):
         class ExampleSerializer(serializers.Serializer):
             omitted = serializers.CharField(required=False, allow_null=True)
+            ommited_read_only = serializers.CharField(
+                required=False,
+                read_only=True,
```

Comment:
btw, the `required=False` is unnecessary here, as it's implied by `read_only=True`.

Reply:
I can take it out if you prefer